cv32/sim/Common.mk

```diff
@@ -203,7 +203,7 @@ sanity: hello-world
 # We link with our custom crt0.s and syscalls.c against newlib so that we can
 # use the c standard library
 $(CUSTOM_DIR)/$(CUSTOM_PROG).elf: $(CUSTOM_DIR)/$(CUSTOM_PROG).c
-	$(RISCV_EXE_PREFIX)gcc -march=rv32imc -o $@ -w -Os -g -nostdlib \
+	$(RISCV_EXE_PREFIX)gcc -mabi=ilp32 -march=rv32imc -o $@ -w -Os -g -nostdlib \
```

Comment:
What does `-mabi=ilp32` do and why do we need it?   Does it affect other tests?

Reply:
mabi specifies integer and floating-point calling convention.

Without this option gcc will error with the following:
c1: error: requested ABI requires -march to subsume the 'D' extension